folio-spring-system-user/src/test/java/org/folio/spring/systemuser/SystemUserServiceTest.java

```diff
@@ -158,6 +174,19 @@ void authSystemUser_negative_emptyHeaders() {
         .hasMessage("User [username] cannot login with expiry because of missing tokens");
   }
 
+  @Test
+  void overloaded_authSystemUser_negative_emptyHeaders() {
+    when(expectedResponse.getHeaders()).thenReturn(new HttpHeaders());
+
+    var systemUser = systemUserValue();
+
+    var systemUserService = systemUserService(systemUserProperties());
+    assertThatThrownBy(() -> systemUserService
+        .authSystemUser("tenanId", "username", "password"))
+        .isInstanceOf(AuthorizationException.class)
+        .hasMessage("Cannot retrieve okapi token for tenant: tenanId");
```

Comment:
This string is repeated quite a few times and has a public constant in `SystemUserService.CANNOT_RETRIEVE_OKAPI_TOKEN_FOR_TENANT`. I think it would be better to use that constant in each of the tests here.

Reply:
Done. please have a look.